apps/api/v2/src/app.ts

```diff
@@ -1,11 +1,9 @@
-import { getEnv } from "@/env";
 import { HttpExceptionFilter } from "@/filters/http-exception.filter";
 import { PrismaExceptionFilter } from "@/filters/prisma-exception.filter";
-import { SentryFilter } from "@/filters/sentry-exception.filter";
```

Comment:
@ThyMinimalDev Was there any specific reason this was added that setupNestErrorHandler doesn't handle?

Reply:
no I think it was more of a fail on our side for not knowing setupNestErrorHandler exists, good one